### ISSUE TITLE
Should we remove downgrade?

### ISSUE THREAD

**Issue by odunbar** [CliMA/ClimaRivers.jl#29]
## Issue 
downgrade always seems to fail due to the fact that dependent packages do not need to satisfy this kind of compatability in their own dependencies, Is it useful to have around?

**Comment by charleskawczynski** [comment]
I think that means that the compat entries in ClimaRivers needs to be updated so that they do pass.

**Comment by charleskawczynski** [comment]
Unfortunately, there's not yet an easy way to figure out what those compats should be, but I know that at least one julia developer is working on it.

**Comment by odunbar** [comment]
So I did manage to resolve downgrade, so maybe we can keep it for now. But it is really not obvious why the solve worked.

**Comment by charleskawczynski** [comment]
Yeah, it's not obvious to me either. My hope is that work on this front results in a "reverse-resolver", that helps you fix your lower compat bounds automatically.
